lib/doorkeeper-mongodb/mixins/mongoid/application_mixin.rb

```diff
@@ -24,26 +24,30 @@ module ApplicationMixin
           validates :name, :secret, :uid, presence: true
           validates :uid, uniqueness: true
           validates :redirect_uri, redirect_uri: true
+          validates :confidential, inclusion: { in: [true, false] }
```

Comment:
Note:

Doorkeeper 5.x doesn't check for confidential support, it just assumes it, so for this PR, I removed the alias to `confidential?` and the other methods that check if it is available.